tools/env-loader/cmd/env-loader.go

```diff
@@ -0,0 +1,101 @@
+/*
+ *  Copyright 2023 Gravitational, Inc
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package main
+
+import (
+	"fmt"
+	"log"
+	"maps"
+	"slices"
+
+	"github.com/alecthomas/kingpin/v2"
+	envloader "github.com/gravitational/shared-workflows/tools/env-loader/pkg"
+	"github.com/gravitational/shared-workflows/tools/env-loader/pkg/writers"
+	"github.com/gravitational/trace"
+)
+
+const ENV_VAR_PREFIX = "ENV_LOADER_"
+
+type config struct {
+	Environment string
+	ValueSet    string
+	Values      []string
+	Writer      string
+}
+
+func parseCli() *config {
```

Comment:
```suggestion
func parseCLI() *config {
```
https://go.dev/wiki/CodeReviewComments#initialisms